.github/workflows/deploy-frontends-to-staging.yml

```diff
@@ -138,7 +145,7 @@ jobs:
           docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-platform-frontend:${{ needs.image-tag.outputs.build_id }}
       - name: Deploy to K8S
         run: |
-          kubectl apply -f k8s/platform/stage-airqo-platform-frontend.yaml
+          kubectl apply -f k8s/netmanager/stage-airqo-platform-frontend.yaml
```

Comment:
Hi @123MwanjeMike , thanks for this amazing work. I sense that things are taking great shape. Brilliant file changes! :) 

Please shed more light on this change in name from platform to netmanager? In summary, what does it mean in the current context where we shall two platform projects running at the same time?

Reply:
The old one(netmanager) will be phased out as soon as the new one is available. However, this changes is to allow maintenance works reflect on the old one.